server/common/mail/templates/index.js

```diff
@@ -0,0 +1,18 @@
+'use strict';
+
+const fs = require('fs');
+const path = require('path');
+const pupa = require('pupa');
+const base64Img = require('base64-img');
+const mjml2html = require('mjml');
+const iconPath = '/assets/shopping.png';
+
+function readTemplate(templatePath, params) {
+  const template = fs.readFileSync(path.join(__dirname, templatePath), 'utf8');
```

Comment:
Don't assume paths relative to `__dirname` here, this function **must** receive fully qualified path from outside.